src/Nethermind/Nethermind.Core/Transaction.cs

```diff
@@ -39,6 +40,9 @@ public class Transaction
 
         public UInt256 Nonce { get; set; }
         public UInt256 GasPrice { get; set; }
+
+        [JsonIgnore]
```

Comment:
To remove?

Reply:
Yes. Right now I'm working on WrappedTransaction, GasBottleneck will be removed from Transaction at all